plugins/com.google.cloud.tools.eclipse.usagetracker.test/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsPingManagerTest.java

```diff
@@ -70,6 +76,10 @@
         }
       });
 
+  @Mock IEclipsePreferences preferences;
+  @Mock private Display display;
+  @Spy private ConcurrentLinkedQueue<PingEvent> pingEventQueue;
```

Comment:
Is it expected that the actual implementation of methods is used in the test, or could this be a @Mock too?